projects/storefrontlib/utils/address-number-utils.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Address } from '@spartacus/core';
+
+/**
+ * Get strings for phone and mobile numbers
+ */
+export function getAddressNumbers(
+  address: Address,
+  textPhone: string,
+  textMobile: string
+): string | undefined {
+  let numbers;
+  if (address.cellphone && address.phone) {
+    numbers = `${textPhone}: ${address.phone}\n${textMobile}: ${address.cellphone}`;
+    if (address.cellphone === address.phone) {
+      numbers = textMobile + ': ' + address.cellphone;
+    }
+  } else if (address.cellphone) {
+    numbers = textMobile + ': ' + address.cellphone;
+  } else if (address.phone) {
+    numbers = textPhone + ': ' + address.phone;
+  }
+  return numbers;
```

Comment:
Lets tidy this function up. There is no need to keep reassigning a variable.

```suggestion
  if (address.cellphone && address.phone) {
    if (address.cellphone === address.phone) {
      return textMobile + ': ' + address.cellphone
    }
    return `${textPhone}: ${address.phone}\n${textMobile}: ${address.cellphone}`
  }

  if (address.cellphone) {
    return textMobile + ': ' + address.cellphone
  }

  if (address.phone) {
    return textPhone + ': ' + address.phone
  }

  return
```